src/tests/suite/SConscript

```diff
@@ -117,8 +117,15 @@ def scons():
     dfstest = daos_build.program(newenv, 'dfs_test', c_files + daos_test_tgt,
                                  LIBS=['daos_common'] + libraries)
 
+    c_files = Split("""daos_debug_set_params.c""")
+    newenv.AppendUnique(CPPPATH=[Dir('../../client/dfs').srcnode()])
+    daosdebug = daos_build.program(newenv, 'daos_debug_set_params',
+                                   c_files + daos_test_tgt,
+                                   LIBS=['daos_common'] + libraries)
```

Comment:
This appears wrong on two counts, newenv already has this directory added on L116, but it also doesn't need this directory in the path as far as I can see.

I don't think this new file need cmocka so the right thing to do might be just use remove this and use denv directly
```suggestion
    daosdebug = daos_build.program(denv, 'daos_debug_set_params',
                                   c_files + daos_test_tgt,
                                   LIBS=['daos_common'] + libraries)
```

You might be able to go further and remove the c_files variable entirely, and look at if daos_test_tgt needs to be linked with this new c file or not.